apps/ledger-live-mobile/src/screens/CustomError/index.tsx

```diff
@@ -28,10 +28,19 @@ export default function CustomError({ route }: CustomErrorPropsProps) {
         <Text variant="body" textAlign={"center"}>
           {t("errors.CustomError.description")}
 
-          <Trans
-            i18nKey="errors.CustomError.errorCode"
-            values={{ errorCode: error?.cause.swapCode }}
-          />
+          {error && "cause" in error && error.cause?.swapCode && (
```

Comment:
As `swapCode` is not a boolean I think that it would be more appropriate to use a ternay operator to avoid common pitfalls.

Reply:
Replied above, should be self explanatory why i went with a ternary

Reply:
I have replied above, please re-consider this comment based on what i replied to the previous comment.
Also, we might want to extend this later, and a ternary would become more complicated.